AGENTS.md

```
@@ -0,0 +1,270 @@
+    ```bash
+    make k8s/vald/delete
+    make k3d/delete
+    ```
+
+---
+
+## 6. Implementation Guidelines
+
+### 6.1 Language-Specific Rules (Critical)
+
+#### Go
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)